R/zchunk_LA106.ag_an_NetExp_FAO_R_C_Y.R

```diff
@@ -30,6 +30,9 @@ module_aglu_LA106.ag_an_NetExp_FAO_R_C_Y <- function(command, ...) {
 
     all_data <- list(...)[[1]]
 
+    Exp_t <- GCAM_commodity <- GCAM_region_ID <- Imp_t <- curr_table <- element <- item <-
+      netExp <- scaler <- value <- year <- NULL   # silence package check notes
+
```

Comment:
When you run `R CMD CHECK` (the "check package" option in RStudio) it looks for undefined symbols and prints a NOTE for each. The problem is that dplyr's uses [non-standard evaluation](http://adv-r.had.co.nz/Computing-on-the-language.html). This allows us to have nice clear and clear pipelines without restating `somedataframe$somecolumn` over and over...but the price is that R CMD CHECK can't determine whether the 'bare' column names being used in a pipeline are legitimate or not. As @rplzzz noted on Slack, this can make it hard to see legitimate notes and warnings.

What we're doing here is providing dummy (NULL) definitions for each term that will appear in a pipeline in the following code. This way R CMD CHECK doesn't have a problem with them.